### ISSUE TITLE
Will this work with stable-ts?

### ISSUE THREAD

**Issue by bakermanbrian** [OpenNMT/CTranslate2#1075]
I am currently using this repo for stabilizing my timestamps in Whisper:
https://github.com/jianfch/stable-ts

Will this library still work with stable-ts?

**Comment by guillaumekln** [comment]
No, this will not work at the moment. The main reason is that stable-ts needs the timestamp logits, but they are currently not returned by the CTranslate2 model.

**Comment by bakermanbrian** [comment]
Makes sense -- any thoughts on when/if this might be added?

**Comment by guillaumekln** [comment]
For now I prefer following the improvements and changes made to the "official" Whisper implementation ([openai/whisper](https://github.com/openai/whisper/)). So I don't plan to work on this issue in the short term.

**Comment by ItakeLs** [comment]
Currently working on this, if I figure it out I'll post my solution

**Comment by guillaumekln** [comment]
We added the method `Whisper.align` that can be used to implement the word-level timestamps as implemented in openai/whisper. The method is already used in the [faster-whisper](https://github.com/guillaumekln/faster-whisper) demo.

Additionally, stable-ts [has been updated](https://github.com/jianfch/stable-ts/commit/2248087c9c2fd1ff5e0d10bd9834fd7bfed8162a) to implement the same word timestamps logic so it should now be possible for them to use CTranslate2.
